storage/src/mmr/mem.rs

```diff
@@ -441,216 +465,216 @@ mod tests {
+        "22adf3131035ed1ef9808b658223a4278d5a06efb648577e88114a7b814e65cb",
+        "56d44dbdaa62c1e1f826f1a99a55e4415d99f27043bcc3b999bb79115aa1cf89",
+        "c805263481303944d9189469488d1dd36dbdc6d6d42fdc2971375051dfc511db",
+        "5ebdcd3f7e2fe558be557d799f3ced886fc6a08cf38fa775d713f3f8b9485246",
+        "7da5931e8cbcb6e151f2106e0b5f5198f44e62f2b29a4e47f3e3a4b05e622a36",
+        "7dd08e28700d60231868088da9cbbb46056b732076edf1d6e07b816f9563594e",
+        "b9be5856487edf3e7814e55bec8c118ddb5bef8150fb9ee74b40dae18b512a9e",
+        "2f481eb9eb3ae2556ca6bd38478a7da87af01f0428242173adb086abb3258507",
+        "3d5e3dda5b0c2cdab22ca513b4cebb024a6c260a90da24414e751d7cb0ee68b7",
+        "d67a3388529c932f85daaa03c114dde54661716a33aada66b482337ff816c23e",
+        "d16c6c6ad379b79cf417e1339afa81e63b5be20a6d19dbb9c6a5991cf7f78825",
+        "fb6e7bde608b1bc0f5be4c34a63c51a4b2e934c91057bbd1e83860125d2e917c",
+        "01902ecdab30d33d0df32fd32e663f46ec1aa09b10fbbaa5cca5b9038f635da4",
+        "322428a14306d9cd75a54d7d4ce586c5a62a4beb84929f36703480c87a16baa0",
+        "ae494e35f8632d1e89834750bf6b9f83a5abaf5ae108d387d75a99a41f262f53",
+        "4d020bfff37176bafea76c529dbd743435c95efa64e4f79b613fa6ca11770ad7",
+        "898922cbc36eaff9b38f17dd32fcfd22540d3df4bef31a0b6100660a34893f7b",
+        "7fea96d799e7d286d99d1be11c28d02180d877d48d1d8258787c8e8df5b25467",
+        "ef299195417e6aa3fe07ce847bbb1e4dfab8959700ccac0f0ca6f62d0d78bde5",
+        "85e20d7eeea9600eb4313ba2f821fd25fa32a31f4d522b20bd71a4d4c7d3a9af",
+        "6c748202320e8edc1fc1c5d0929d1824f728a0edfdb6732e59dcf7041e96edb4",
+        "615079f8a57735afd423f355877a3421375dbad5ac485b3ccee503ec14788cec",
+        "bbe47ac28fc3458d21c89b042e75539211e63f55e80f52edaee8365d2ca1d311",
+        "a280b5d71c849e421292e484151e77dc1a0315e443100275a9b4fbded1ab6140",
+        "a2a7815b07c156de0b9f5a2fd0c6ad73bd70d06fef7c328c0fefe95a43899c52",
+        "c5252ffc458658cd274ea080b4c52a84ab0915d7e9fdc949b94bbc462fb46543",
+        "bfa7188cd6e33ceb7db6c87b8afc3cea4098b90293d0d962315cc83d51f4710f",
+        "a2f018444b66aef64c149c1bdf0bec28784772f09cc4abb3dcccb882bde089f8",
+        "27f7d66ded874e6b5ac1843ea8147df16c79ac09fd1db2a5a0f5d6e76dea12e6",
+        "a15e25ef9f2b99281a81a9926cd2b5bc9335ece7c35dd4ad99845bfb0ba8c0e9",
+        "6ef54ff7810f3b28821950a29ec9a797c5f2b3fe79fb50bda7c392777752a4ff",
+        "77a20dc05f10e9b4f494d3d22cdcddb7cbe5e99f951c2cee900c9024f34ab4e8",
     ];
 
     /// Test that the MMR root computation remains stable by comparing against previously computed
     /// roots.
     #[test]
     fn test_root_stability() {
         let mut hasher = Sha256::new();
-        for i in 0..200 {
+        for i in 0u64..200 {
             let mut mmr = Mmr::<Sha256>::new();
-            for _ in 0u64..i {
-                hasher.update(&i.to_be_bytes());
+            for j in 0u64..i {
+                hasher.update(&j.to_be_bytes());
```

Comment:
this is the diff that changed the computed digests. A bug in the test, not in the mmr code.